### PR TITLE
DT-1547,DT-1549,DT-1552,DT-1554,DT-1587: Update Container Positionings

### PR DESCRIPTION
**Pull request by HenrikHL**
[DT-1547](https://dcsa.atlassian.net/browse/DT-1547) and [DT-1549](https://dcsa.atlassian.net/browse/DT-1549) - Remove `empty` from container positioning attributes
[DT-1552](https://dcsa.atlassian.net/browse/DT-1552) and [DT-1554](https://dcsa.atlassian.net/browse/DT-1554) - Merge containerPickup and containerDepotRelease into one optional object
[DT-1587](https://dcsa.atlassian.net/browse/DT-1587) - Update description

[DT-1547]: https://dcsa.atlassian.net/browse/DT-1547?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1549]: https://dcsa.atlassian.net/browse/DT-1549?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1552]: https://dcsa.atlassian.net/browse/DT-1552?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1554]: https://dcsa.atlassian.net/browse/DT-1554?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1587]: https://dcsa.atlassian.net/browse/DT-1587?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: bkg/v2/BKG_v2.0.0.yaml [hunk 1]
```diff
@@ -6959,22 +6959,30 @@ components:
...
+            
+            **Condition:** Only applicable to merchange haulage service at origin (`Receipt type at origin = 'CY'`).
+          properties:
+            dateTime:
+              type: string
+              format: date-time
+              description: |
+                The date and time requested by the shipper for the pick-up of the empty container(s) at the `Empty Container Depot Release Location` (DRL), if provided.
+              example: '2024-07-05T09:03:00-02:00'
+            location:
```
Comment (palatsangeetha) [inline bkg/v2/BKG_v2.0.0.yaml:6984]: Should this be depotReleaseLocation or location?

File: bkg/v2/BKG_v2.0.0.yaml [hunk 2]
```diff
@@ -8116,14 +8130,14 @@ components:
           description: |
             Number of confirmed equipment units
           example: 3
-        emptyContainerPositionings:
+        containerPositionings:
           type: array
           description: |
-            List of `ECP` (Empty Container Positioning) locations.
+            List of `CPO` (Container Positioning) locations.
```
Comment (palatsangeetha) [inline bkg/v2/BKG_v2.0.0.yaml:8136]: Shouldn't be the description as same as IFS one?

File: bkg/v2/BKG_v2.0.0.yaml (hunk 1 above)
Comment (HenrikHL) [reply bkg/v2/BKG_v2.0.0.yaml:6984]: Text is synced with issue now

File: bkg/v2/BKG_v2.0.0.yaml (hunk 2 above)
Comment (HenrikHL) [reply bkg/v2/BKG_v2.0.0.yaml:8136]: Text is synced with the Requested Equipment + added that it is estimated date time

File: bkg/v2/BKG_v2.0.0.yaml [hunk 3]
```diff
@@ -7048,22 +7056,30 @@ components:
...
+            
+            **Condition:** Only applicable to merchange haulage service at origin (`Receipt type at origin = 'CY'`).
+          properties:
+            dateTime:
+              type: string
+              format: date-time
+              description: |
+                The date and time requested by the shipper for the pick-up of the empty container(s) at the `Empty Container Depot Release Location` (DRL), if provided.
+              example: '2024-07-05T09:03:00-02:00'
+            location:
```
Comment (palatsangeetha) [inline bkg/v2/BKG_v2.0.0.yaml:7081]: The propertName - depotReleaseLocation as per jira issue

File: bkg/v2/BKG_v2.0.0.yaml [hunk 4]
```diff
@@ -7160,52 +7176,50 @@ components:
...
+          $ref: '#/components/schemas/ContainerPositioningLocation'
       required:
-        - emptyContainerPositioningLocation
+        - location
 
-    EmptyContainerPositioningEstimated:
+    ContainerPositioningEstimated:
       type: object
-      title: Empty Container Positioning
+      title: Container Positioning
```
Comment (palatsangeetha) [inline bkg/v2/BKG_v2.0.0.yaml:7198]: The title - Container Positioning Estimated

File: bkg/v2/BKG_v2.0.0.yaml (hunk 3 above)
Comment (HenrikHL) [reply bkg/v2/BKG_v2.0.0.yaml]: Fixed

File: bkg/v2/BKG_v2.0.0.yaml (hunk 4 above)
Comment (HenrikHL) [reply bkg/v2/BKG_v2.0.0.yaml]: Fixed

**Comment by HenrikHL** [comment]
Will merge now - any changes will have to come in a fix in next snapshot

**Pull request merged**